homeassistant/components/peco/sensor.py

```diff
@@ -23,16 +27,44 @@
 
 from .const import CONF_COUNTY, DOMAIN, LOGGER, SCAN_INTERVAL
 
+
+@dataclass
+class PECOSensorEntityDescriptionMixin:
```

Comment:
Why the Mixin when all `PecoSensorEntityDescription`s use the same function? Wouldn't this work just directly in `PecoSensorEntityDescription`?

Reply:
@farmio It is for making `value_fn` required (non-optional).

You can't inherit a required field into a dataclass.

Reply:
Ha, didn't know that. Thanks 👍